### PR TITLE
[fix bug 1370730] Add firefox/features/memory page

### PR DESCRIPTION
**Pull request by alexgibson**
## Description
- Adds /firefox/features/memory/ page (en-US only for now, like the other hub pages)
- Adds new thumbnail image to /firefox/features/ landing page.

Note: still waiting on a link, plus meta info for the page.

This needs to go out by **Tuesday 6/13**

## Bugzilla link
https://bugzilla.mozilla.org/show_bug.cgi?id=1370730

## Testing
-  Make sure no typo's?

## Checklist
- [ ] Requires l10n changes.
- [x] Related functional & integration tests passing.

### CODE REVIEW

File: bedrock/firefox/templates/firefox/features/memory.html [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+{% block page_title_suffix %}{% endblock %}
+{% block page_desc %}{{ _('If your web browser uses too much memory, switch to Firefox') }}{% endblock %}
+{% block page_image %}{{ static('img/firefox/features/thumbnails/memory.png') }}{% endblock %}
+
+{% block page_css %}
+  {{ super() }}
+  {% stylesheet 'firefox-features-hub-detail' %}
+{% endblock %}
+
+{% block body_id %}firefox-features-fast{% endblock %}
```
Comment (jpetto) [inline bedrock/firefox/templates/firefox/features/memory.html]: Nit - should be `firefox-features-memory`? (Probably doesn't really matter...)

File: bedrock/firefox/templates/firefox/features/memory.html [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+          memory to let you browse smoothly and leaving plenty of memory to keep your
+          computer responsive.
+        {% endtrans %}
+        </p>
+      </li>
+      <li class="features-list-item">
+        <img src="{{ static('img/firefox/features/thumbnails/memory.png') }}" alt="">
+        <h3>{{ _('Stop running out of memory') }}</h3>
+        <p>
+        {% trans url="#TODO"%}
```
Comment (jpetto) [inline bedrock/firefox/templates/firefox/features/memory.html]: Noting the `#TODO`

File: bedrock/firefox/templates/firefox/features/memory.html [hunk 3]
```diff
@@ -0,0 +1,125 @@
...
+    <h2 class="section-title"><span>{{ _('Features') }}</span></h2>
+
+    <ul class="features-list">
+      <li class="features-list-item">
+        <img src="{{ static('img/firefox/features/thumbnails/memory/fast.png') }}" alt="">
+        <h3>{{ _('Speed up your computer') }}</h3>
+        <p>
+        {% trans %}
+          Every computer program you run takes up some memory. When too much is used,
+          your system can  slooooowww down. Firefox aims for a balance—using enough
```
Comment (jpetto) [inline bedrock/firefox/templates/firefox/features/memory.html]: The lowest of nits - an extra space.

Do we care about using `&mdash;` for the dash here?

**Comment by alexgibson** [comment]
@jpetto updated thanks - added correct page title/description so this is now good to go.

**Pull request merged**